src/object/rigid_body.rs

```diff
@@ -272,7 +278,8 @@ impl<N: Scalar> RigidBody<N> {
     pub fn new(shape:           Arc<Box<Repr<Point<N>, Matrix<N>>>>,
                mass_properties: Option<(N, Point<N>, AngularInertia<N>)>,
                restitution:     N,
-               friction:        N)
+               friction:        N,
+               groups:          RigidBodyCollisionGroups)
```

Comment:
That one should not have `groups` on its constructor either. On this method, the static or dynamic status of the rigid body is automatically deduced from `mass_properties`. If it is set to `None`, then the rigid body is static. It is dynamic otherwise.